exercises/grains/example.rb

```diff
@@ -1,9 +1,14 @@
+module BookKeeping
+  VERSION = 1
+end
+
 module Grains
   def self.square(number)
+    fail ArgumentError if number <= 0 || number > 64
     2**(number - 1)
   end
 
   def self.total
-    square(65) - 1
+    2**64 - 1
```

Comment:
Because of the change to the square method, calling `square(65)` would raise an `ArgumentError`, so I just made the smallest change possible to make the tests pass. Is this okay?